Sources/ContainerClient/Core/ClientContainer.swift

```diff
@@ -145,16 +137,67 @@ extension ClientContainer {
 
 extension ClientContainer {
     public func bootstrap(stdio: [FileHandle?]) async throws -> ClientProcess {
-        let client = self.sandboxClient
-        try await client.bootstrap(stdio: stdio)
-        return ClientProcessImpl(containerId: self.id, client: self.sandboxClient)
+        let request = XPCMessage(route: .containerBootstrap)
+        let client = Self.newClient()
+
+        for (i, h) in stdio.enumerated() {
+            let key: XPCKeys = {
+                switch i {
+                case 0: .stdin
+                case 1: .stdout
+                case 2: .stderr
+                default:
+                    fatalError("invalid fd \(i)")
```

Comment:
If a bug accidentally sends a longer array, this would crash the whole program. Consider a typed error instead

Reply:
Yea this was a copy paste from the type we already had, I'd love to fix this up now though. I've been trying to get rid of some of our fatalError's in CZ recently too